### DISCUSSION TITLE
Active devices

### DISCUSSION THREAD

**Discussion by pdifeo** [rustdesk/rustdesk#5835] [General]
In the device list from the console it would be useful to have a search filter on active and inactive devices. I mean devices that are online and connected to the server. Now you only see the blue or red dot, but no possibility to filter for their status.

**Comment by kess78** [comment]
+1 Absolutely, this is a must !

**Comment by rustdesk** [comment]
Online status is sortable now on 1.5.0 https://www.reddit.com/r/rustdesk/comments/1isg7zs/selfhost_server_pro_150_is_ready/

**Reply by rustdesk** [reply]
![image](https://github.com/user-attachments/assets/75ac23d1-6e82-44ad-9e08-25ef2ded87c8)

**Comment by kess78** [comment]
Sorry, but I thought this was the RustDesk Client discussion. For that reason I voted that feature, because it's not yet available in the RD client

**Comment by pdifeo** [comment]
The descriptions are not visible with ligth theme. For me not is a problem, only for inform you

![image](https://github.com/user-attachments/assets/56c0eba7-881f-4cc3-807f-67fac871e52e)

**Reply by rustdesk** [reply]
Got it, will fix today

**Reply by rustdesk** [reply]
@vitoway

**Comment by kess78** [comment]
Thank you, but what about the same feature implemented in the client ?
And perhaps it would be better a filter, not only a sortable list...
